RELEASE-NOTES.txt

```diff
@@ -1,6 +1,11 @@
 Unreleased
 ------
 
+1.58.1
+------
+* [*] Global styles: Check for undefined values and merge user colors [https://github.com/WordPress/gutenberg/pull/33707]
```

Comment:
Could we add an iOS only indicator? It wasn't immediately obvious to me that this wasn't testable from Android.

Reply:
Yeah, good point! I'll update it.

Reply:
FYI this fix was needed for both platforms, but on WordPress Android is behind a feature flag =) so I'd say we should keep it. For the main apps release notes, it'll be included only for iOS.

Reply:
I'll update it once the last PR checks pass so we know that the optional tests are in good shape.

Reply:
Got it, my main worry was about it accidentally going in the Android app's release notes and causing confusion for testers/editorial. If that's not going to happen then I agree it's not necessarily worth updating. :)

Reply:
Yeah, that's true, let's make sure we don't update the Release notes on the main apps for the global styles fix, since it was already included for iOS only 😃

Thank you for double-checking!

Reply:
> FYI this fix was needed for both platforms, but on WordPress Android is behind a feature flag =) so I'd say we should keep it. For the main apps release notes, it'll be included only for iOS.

Ok, I see in the 17.9 release notes that is only referenced in the WPiOS version 👍 .

[WPAndroid release notes](https://github.com/wordpress-mobile/WordPress-Android/blob/6a4cdce25a90ed96ef5da3633071505fe6b008bb/RELEASE-NOTES.txt#L7-L10):
```
17.9
-----
* [***] Block editor: New Block: Embed block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3727]
* [*] Added webp image format support [https://github.com/wordpress-mobile/WordPress-Android/pull/15068]
```

[WPiOS release notes](https://github.com/wordpress-mobile/WordPress-iOS/blob/7dc399e47194a4c82439e55cfb330425227e165c/RELEASE-NOTES.txt#L5-L10):
```
17.9
-----
* [internal] Redirect Terms and service to open the page in an external web view [#16907]
* [internal] Converted Comment model methods to Swift. Should be no functional changes, but could cause regressions. [#16898, #16905, #16908, #16913]
* [*] Enables Support for Global Style Colors with Full Site Editing Themes [#16823]
* [***] Block editor: New Block: Embed block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3727]
```

Reply:
Great, thanks for the clarification, both!